RHEL/6/input/profiles/ftp-server.xml

```diff
@@ -1,5 +1,5 @@
 <Profile id="ftp-server" extends="server">
-<title override="true">ftp</title>
+<title override="true">FTP Server Profile</title>
 <description override="true">This profile is for FTP servers.</description>
```

Comment:
Can we use this as an opportunity to call out  vsftpd, vs any/all FTP servers?

e.g. 

title:  ``FTP Server Profile`` --> ``FTP Server Profile (vsftpd) ``
descr: ``This is a profile for FTP servers`` --> ``This is a profile for the vsftpd FTP server``

IMHO this is not worth holding up a merge (happy to merge once jenkins passes), but it'd be great to get this updated while this part of the code is being touched :)